hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -90,6 +90,12 @@
       .withDocumentation("When set to true, compaction service is triggered after each write. While being "
           + " simpler operationally, this adds extra latency on the write path.");
 
+  public static final ConfigProperty<String> SCHEDULE_ASYNC_COMPACT = ConfigProperty
+      .key("hoodie.compact.schedule.async")
+      .defaultValue("false")
```

Comment:
`COMPACT_SCHEDULE_ASYNC` seems  better.

Reply:
`COMPACT_SCHEDULE_INLINE` please to keep inline with the string options sequence.